tests/quaternion.rs

```diff
@@ -113,6 +113,36 @@ mod from {
     }
 }
 
+mod arc {
+    use cgmath::*;
+
+    fn test(src: Vector3<f32>, dst: Vector3<f32>) {
+        let q = Quaternion::from_arc(src, dst, None);
+        let v = q.rotate_vector(src);
+        assert_approx_eq!(v, dst);
+    }
```

Comment:
Inline as in put the code in `test_opposite`, rather than in a separate function ^_^